dotCMS/src/main/java/com/dotmarketing/servlets/BinaryExporterServlet.java

```diff
@@ -576,7 +578,13 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws Servl
 			if(!resp.isCommitted()){
 			  resp.sendError(HttpServletResponse.SC_NOT_FOUND);
 			}
-		}catch (Exception e) {
+		} catch (IOException e) {
+            Logger.debug(BinaryExporterServlet.class, e.getMessage(),e);
+            Logger.error(BinaryExporterServlet.class, e.getMessage());
+            if(!resp.isCommitted()){
+              resp.sendError(HttpServletResponse.SC_NOT_FOUND);

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a "try/catch" block for "sendError". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1989)